cardano-cli/test/cardano-cli-golden/Test/Golden/Governance/Committee.hs

```diff
@@ -187,13 +196,13 @@ hprop_golden_governanceUpdateCommittee =
 hprop_golden_governance_committee_cold_extended_key_signing :: Property
 hprop_golden_governance_committee_cold_extended_key_signing =
   propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
-    skeyFile <- noteInputFile "test/cardano-cli-golden/files/input/governance/committee/cc.extended.cold.skey"
+    void $ noteInputFile "test/cardano-cli-golden/files/input/governance/committee/cc.extended.cold.skey"
     txBody <- noteInputFile "test/cardano-cli-golden/files/input/governance/drep/extended-key-signing/tx.body"
 
     outGold <- H.note "test/cardano-cli-golden/files/golden/governance/committee/tx.cold.extended.signed"
     outFile <- H.noteTempFile tempDir "outFile"
 
-    H.noteM_ $ execCardanoCLI
+    H.noteM_ $ bracketSem extendedColdCCSkeySem $ \skeyFile -> execCardanoCLI
```

Comment:
Semaphore for input file (which is only read) seems redundant here.

Reply:
Changed :+1: